app/services/applications/change_funding_eligibility.rb

```diff
@@ -49,5 +53,15 @@ def validate_funding_eligiblity_status_with_funded_place
     def declared_as_billable_or_changeable?
       application.declarations.billable_or_changeable.count.positive?
     end
+
+    def send_eligible_for_funding_email
+      ApplicationFundingEligibilityMailer.eligible_for_funding_mail(
+        to: application.user.email,
+        full_name: application.user.full_name,
+        provider_name: application.lead_provider.name,
+        course_name: application.course.name,
+        ecf_id: application.ecf_id,
+      ).deliver_now
```

Comment:
`.deliver_later` ? That way if there's flakiness connecting to Notify this will still get delivered rather than the action failing?